### PR TITLE
Html renderer node

### PR DESCRIPTION
**Pull request by fakob**
Added an html render node which allows to embed any iframe code (youtube, vimeo, website). Unfortunately we can not embed our github page, as it disallows embedding.

Screen Shot 2022-10-11 at 22 17 11

### CODE REVIEW

File: src/nodes/draw/html.tsx [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+import { CustomArgs } from '../../utils/interfaces';
+import { SOCKET_TYPE, customTheme } from '../../utils/constants';
+import HybridNode from '../../classes/HybridNode';
+
+const inputSocketName = 'Html';
+
+export class HtmlRenderer extends HybridNode {
+  update: (newHeight?) => void;
+
+  protected getIsHybrid(): boolean {
```
Comment (magnificus) [inline src/nodes/draw/html.tsx:20]: this function should not be needed anymore I think

File: src/nodes/draw/html.tsx [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+      nodeHeight,
+      minNodeWidth: nodeWidth / 2,
+      minNodeHeight: nodeHeight / 2,
+    });
+
+    if (customArgs?.initialData) {
+      this.setInputData(inputSocketName, customArgs?.initialData);
+    }
+
+    // when the Node is added, create the container and react component
```
Comment (magnificus) [inline src/nodes/draw/html.tsx:82]: could we have this in the common hybrid node class? It seems to be roughly the same for all hybrid nodes

**Review by magnificus — APPROVED** [review]

File: src/nodes/draw/html.tsx (hunk 1 above)
Comment (fakob) [reply src/nodes/draw/html.tsx:20]: Ah, yes. Good catch.

Comment (fakob) [reply src/nodes/draw/html.tsx]: Updated

File: src/nodes/draw/html.tsx (hunk 2 above)
Comment (fakob) [reply src/nodes/draw/html.tsx:77]: To be honest, I really do not know how to do this. I have to pass in the `ParentComponent` and `props` which are custom for every hybrid node. And this has to happen after the node has been created.
Do you want to give it a shot?

Comment (magnificus) [reply src/nodes/draw/html.tsx:77]: Then I would make a function in hybridnode to be overridden which returns this parentcomponent and props, I can take a look if you want

Comment (fakob) [reply src/nodes/draw/html.tsx:77]: Yes, please do. Do you want to push to this branch or should I merge and you create a separate PR?

Comment (magnificus) [reply src/nodes/draw/html.tsx:77]: I can do it on this as long as i dont forget

**Pull request merged**